spec/manuals_publisher/removing_content_spec.rb

```diff
@@ -0,0 +1,59 @@
+feature "Removing content on Manuals Publisher", manuals_publisher: true do
+  include ManualsHelpers
+
+  let(:title) { title_with_timestamp }
+  let(:section_title) { title_with_timestamp }
+  let(:section_slug) { section_title.downcase.tr(" ", "-").gsub(/[^\w-]/, '') }
+
+  scenario "Remove a section from a published manual" do
+    given_there_is_a_published_manual_with_sections
+    when_i_remove_a_section_and_ignore_the_500
+    and_i_publish_the_manual
+    then_the_removed_section_redirects_to_the_root_page
+  end
+
+  def given_there_is_a_published_manual_with_sections
+    create_draft_manual(title: title)
+    @edit_manual_url = current_url
+
+    create_manual_section
+    create_manual_section(title: section_title)
+    publish_manual
+
+    contents_url = find_link("View on website")[:href]
+    reload_url_until_status_code(contents_url, 200)
+    reload_url_until_status_code(removed_section_url, 200)
+  end
+
+  def when_i_remove_a_section_and_ignore_the_500
+    click_link section_title
+    click_link "Withdraw section"
+
+    fill_in "Change note", with: "Test removal"
+    click_button "Yes"
+
+    visit @edit_manual_url
```

Comment:
Is this the point where we do something special because of the 500?

Could add a comment such as "We manually visit this URL due to submitting this form causing a 500. See: https://trello.com/c/hpkfiCU5/54-withdrawing-a-section-of-a-published-manual-returns-a-500 for the issue - If this issue is now resolved this should be changed to <insert what it should be>"